### ISSUE TITLE
resourceContainerizerMode JS Client-Module split issue

### ISSUE THREAD

**Issue by BluePeer** [altmp/altv-issues#2368]
### Description of the problem

if you have a js file, bigger then the split size (the case i have)
10MB js file (dump of animation list)
then the js file can't be read in if  splitmode is active
```
[05:35:38][js] [V8] Starting script client/index.js
[05:35:38][Error][js] [V8] Exception at bluerpclient:client/anims.js:158682
[05:35:38][Error][js]               "xm_export const ANIM_DICTS = [
[05:35:38][Error][js]   SyntaxError: Invalid or unexpected token
    at <bootstrapper>:18:19
[05:35:38][Error][js] SyntaxError: Invalid or unexpected token
[05:35:38][Error][js] [V8] v8::Module::GetModuleNamespace: v8::Module::GetModuleNamespace must be used on an instantiated module
[05:35:38][js] [V8] Started script client/index.js
```
the issue in this is in the ~ middle of the file where the split is ( resourceBlockSize = 5)

### Reproduction steps

create a js client resource, that contains a JS file larger then the splitsize (resourceBlockSize)

### Expected behaviour

not destroy files on splitpoint

### Operating system

linux

### Version

16.2.24

### Confirmation of issue's presence

- [X] *By submitting this ticket, I affirm that I have verified the presence of this issue on the latest RC (Release Candidate) version available at the time of writing this ticket.*

**Comment by BluePeer** [comment]
[demo.zip](https://github.com/user-attachments/files/17825927/demo.zip)
example resource
used linux

resourceContainerizerMode = true
resourceBlockSize = 5

**Comment by Doxoh** [comment]
fixed with  16.2.25-rc.3
